package.json

```diff
@@ -17,7 +17,8 @@
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "test": "gulp ci"
+    "test": "gulp ci",
+    "bootstrap": "npm install -g bower gulp jison"
```

Comment:
"nothing" means I have no Git, I have no NPM, I have no Gulp. All I have is a directory and a terminal. Also, this should cover Windows and Linux.